### ISSUE TITLE
Incorrect "variable is not accessed" messages

### ISSUE THREAD

**Issue by roveo** [microsoft/pylance-release#719]
## Environment data

-   Language Server version: v2020.12.1
-   OS and version: Mac OS Mojave 10.14.6
-   Python version: 3.7.5 with pyenv

## Expected behaviour

When a variable is passed to a function/method that Pylance couldn't introspect (sometimes it's not possible for various reasons), it should not be considered unused.

## Actual behaviour

It's grayed out with "`variable` is not used" message on hover.

## Code Snippet / Additional information

I think the best thing I can do here is to show an image of actual code. `Stream.from_postgres_increment` is a class method, but it's nowhere to be found in `Stream` class definition: it's actually loaded later on via entry_points mechanism. Pylance can't run introspection on it (and that's totally OK), but the problem is that it also counts all variables passed to this method as unused, while they are obviously used.

![image](https://user-images.githubusercontent.com/5427958/101897530-bebe5580-3bbb-11eb-93a5-f437eee5a8c3.png)

I don't think there's anything actually "breaking" here, it's more of a logic problem, so I skipped the logs. Will add if you need them.

**Comment by erictraut** [comment]
Could you provide a self-contained, minimal code sample that reproduces the problem you're seeing? I don't have enough context from the screen shot above to repro the problem.

**Comment by roveo** [comment]
I played around a bit and I think it has something to do with how the package was installed.

First, install a package through pip+git:

```
pip install git+https://github.com/python-streamz/streamz.git
```

Then, create this file:

```py
from streamz import Stream


def test(x=0):
    y = 1
    Stream.nomethod(x=x, y=y)
```

`x` and `y` should be gray. If I do the same but install from PyPi (`pip install streamz`), everything works as expected.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.12.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#2020122-11-december-2020
